package.json

```diff
@@ -63,43 +64,46 @@
     "license:check": "npm run license:list -- --failOn `cat .licensebanned | grep '^[^#;]' | awk 'BEGIN { ORS=\"\" } { print p$0\";\"; } END { print \n }'`"
   },
   "dependencies": {
+    "@hapi/basic": "5.1.0",
+    "@hapi/boom": "7.4.2",
+    "@hapi/good": "8.2.0",
+    "@hapi/hapi": "18.3.1",
+    "@hapi/inert": "5.2.1",
+    "@hapi/joi": "15.1.0",
+    "@hapi/joi-date": "1.3.0",
+    "@hapi/vision": "5.5.2",
     "@mojaloop/central-services-auth": "5.2.1",
     "@mojaloop/central-services-database": "5.2.1",
     "@mojaloop/central-services-error-handling": "5.2.0",
     "@mojaloop/central-services-metrics": "5.2.0",
-    "@mojaloop/central-services-shared": "5.2.0",
+    "@mojaloop/central-services-shared": "6.4.1",
+    "@mojaloop/central-services-health": "7.0.0",
     "@mojaloop/central-services-stream": "6.2.2",
     "@mojaloop/forensic-logging-client": "5.2.0",
     "@now-ims/hapi-now-auth": "2.0.0",
     "blipp": "4.0.0",
     "bluebird": "3.5.5",
-    "@hapi/boom": "7.4.2",
     "catbox": "10.0.6",
     "catbox-memory": "4.0.1",
     "commander": "2.20.0",
     "docdash": "1.1.1",
     "flat": "4.1.0",
     "glob": "7.1.4",
-    "@hapi/good": "8.2.0",
-    "@hapi/hapi": "18.3.1",
-    "@hapi/basic": "5.1.0",
     "hapi-auth-bearer-token": "6.2.1",
     "hapi-swagger": "10.0.2",
     "immutable": "3.8.2",
-    "@hapi/inert": "5.2.1",
-    "@hapi/joi": "15.1.0",
     "joi-currency-code": "2.0.2",
-    "@hapi/joi-date": "1.3.0",
     "jsdoc": "3.6.2",
     "knex": "0.17.6",
     "lodash": "4.17.11",
     "mustache": "3.0.1",
     "raw-body": "2.4.1",
     "rc": "1.2.8",
     "request": "2.88.0",
+    "request-promise": "4.2.4",
```

Comment:
i would suggest using axios as it is used in other APIs. If you feel that request-promise-native gives you a more desired effect then it is fine

Reply:
I'll check it out. This was just what I'm most comfortable using.